sdk/keyvault/azure-security-keyvault-certificates/src/certificate_client.cpp

```diff
@@ -1,497 +1,497 @@
-  auto value = DeletedCertificatesPagedResponseSerializer::Deserialize(*rawResponse);
-  return DeletedCertificatesPagedResponse(
-      std::move(value), std::move(rawResponse), std::make_unique<CertificateClient>(*this));
-}
-
-Azure::Response<KeyVaultCertificateWithPolicy> CertificateClient::ImportCertificate(
-    std::string const& certificateName,
-    ImportCertificateOptions const& options,
-    Azure::Core::Context const& context) const
-{
-  auto payload = ImportCertificateOptionsSerializer::Serialize(options);
-  Azure::Core::IO::MemoryBodyStream payloadStream(
-      reinterpret_cast<const uint8_t*>(payload.data()), payload.size());
-
-  auto request = CreateRequest(
-      HttpMethod::Post, {CertificatesPath, certificateName, ImportPath}, &payloadStream);
-
-  auto rawResponse = SendRequest(request, context);
-  auto value = KeyVaultCertificateSerializer::Deserialize(certificateName, *rawResponse);
-  return Azure::Response<KeyVaultCertificateWithPolicy>(std::move(value), std::move(rawResponse));
-}
-
-Azure::Response<KeyVaultCertificateWithPolicy> CertificateClient::MergeCertificate(
-    std::string const& certificateName,
-    MergeCertificateOptions const& options,
-    Azure::Core::Context const& context) const
-{
-  auto payload = MergeCertificateOptionsSerializer::Serialize(options);
-  Azure::Core::IO::MemoryBodyStream payloadStream(
-      reinterpret_cast<const uint8_t*>(payload.data()), payload.size());
-
-  auto request = CreateRequest(
-      HttpMethod::Post,
-      {CertificatesPath, certificateName, PendingPath, MergePath},
-      &payloadStream);
-
-  auto rawResponse = SendRequest(request, context);
-  auto value = KeyVaultCertificateSerializer::Deserialize(certificateName, *rawResponse);
-  return Azure::Response<KeyVaultCertificateWithPolicy>(std::move(value), std::move(rawResponse));
-}
-
-Azure::Response<KeyVaultCertificate> CertificateClient::UpdateCertificateProperties(
-    std::string const& certificateName,
-    std::string const& certificateVersion,
-    CertificateProperties const& certificateProperties,
-    Azure::Core::Context const& context) const
-{
-  auto payload = CertificateUpdateOptionsSerializer::Serialize(certificateProperties);
-  Azure::Core::IO::MemoryBodyStream payloadStream(
-      reinterpret_cast<const uint8_t*>(payload.data()), payload.size());
-
-  auto request = CreateRequest(
-      HttpMethod::Patch, {CertificatesPath, certificateName, certificateVersion}, &payloadStream);
-
-  auto rawResponse = SendRequest(request, context);
-  auto value = KeyVaultCertificateSerializer::Deserialize(certificateName, *rawResponse);
-  return Azure::Response<KeyVaultCertificate>(std::move(value), std::move(rawResponse));
-}
-
-const ServiceVersion ServiceVersion::V7_2("7.2");
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/keyvault/certificates/certificate_client.hpp"
+
+#include "azure/keyvault/shared/keyvault_shared.hpp"

```

Comment:
The changes to this file is only a header re-order. It's not relevant to this PR. Can you revert the change to this file?